use-cases/client-server-msgs-perf/svmsg_file_server.c

```diff
@@ -123,6 +131,16 @@ Client_info ActiveClients[MAX_CLIENTS];
 static unsigned int   NumActiveClientsHWM = 0;
 static unsigned int   NumActiveClients    = 0;
 
+/**
+ * On MacOSX, only the CLOCK_THREAD_CPUTIME_ID clock has a resolution of 1ns.
+ * On Linux, all clocks seem to have 1 ns resolution, so use the CLOCK_REALTIME
+ * clock as the default clock to use. This way, the throughput metric computed
+ * as (nops/sec) is consistent with normal expectation of 'sec' as real time
+ * seconds.
+ */
+int Clock_id = CLOCK_REALTIME;
```

Comment:
This variable used to be in `main()` but needs to be referenced by child threads.

So made it global and renamed to Upper-case, to keep a convention I'm maintaining of naming global variables with 1st char upper-case.